index.html

```diff
@@ -12,5 +12,69 @@
 <body>
     <script src="./src/index.js"></script>
     <script src="./node_modules/axios/dist/axios.min.js"></script>
+    <div id="title">
+    <h2>✤✨Weather in✨✤ <div id="city">Name</div></h2>
+    </div>
+    <div class="snowflakes">
+        <div class="snowflake">
+        <img src="/assets/sailormoon.png"></img>
+        </div>
+        <div class="snowflake">
+        <img src="/assets/anyacute.png"></img>
+        </div>
+        <div class="snowflake">
+        💝
+        </div>
+        <div class="snowflake">
+        💝
+        </div>
+        <div class="snowflake">
+        <img src="/assets/sailorcheer.png"></img>
+        </div>
+        <div class="snowflake">
+        💝
+        </div>
+        <div class="snowflake">
+            <img src="/assets/sleepymochi.png"></img>
+        </div>
+        <div class="snowflake">
+            💝
+        </div>
+        <div class="snowflake">
+            <img src="/assets/babymochi.png"></img>
+        </div>
+        <div class="snowflake">
+            💝
+        </div>
+        <div class="snowflake">
+            💝
+        </div>
+        <div class="snowflake">
+            <img src="/assets/goblinanya.png"></img>
+        </div>
+    </div>
+    <section class="weather-box">
+        <div class="temp-controls">
+        <span id="temp-button-up"><img id="arrowleft" src="assets/leftarrow.jpeg"></span> <button id="real-time-temp">72</button><span id="temp-button-down"><img id="arrowright" src="assets/rightarrow.jpeg"></span> 
```

Comment:
In general, folks who use languages (like English) that are left-to-right also interpret a right arrow as "increasing" and a left arrow as "decreasing". Same for time; folks expect time to increase to the right. (The opposite is usually true for right-to-left languages like Arabic.)

Reply:
It was really hard to tell that the number was a button; I had to look at the HTML to know. 😅